content/en/synthetics/guide/app-that-requires-login.md

```diff
@@ -0,0 +1,69 @@
+
+{{< img src="synthetics/guide/app_that_requires_login/incognito.mp4" video="true" alt="Demo of recording a login in incognito">}}
+
+Opening a pop up in incognito mode allows you to start your test's recording from the start URL set in your test configuration with a session completely isolated from your own browser's main session and user data. The freshly opened incognito pop up ignores all your previous browser history: cookies, local data, etc. You are automatically logged out from your account and can start recording your login steps as if you were visiting your website for the first time.
+
+**Note:** Use [the subtest feature][2] to group your login steps into a single subtest that you can then reuse across any other browser tests that require a login.
+
+### SSO login
+
+If your website uses SSO for login, input your application's URL as the starting URL of your browser test. The test performs the required redirections as part of the first default "Navigate to URL" step.
+
+Some SSO providers might detect our browser tests as bots and prevent them from logging in, for example, by adding a re-Captcha. If that is your case, consider reaching out to your SSO provider to see if it is possible to turn off bot detection for a specific set of credentials for testing purposes.
+
+An alternative would be to use a non-SSO approach and leverage a regular username and password combination to go through login.
+
+### Multi-factor authentication
+
+Browser tests can reproduce any actions a regular user can take inside their Chrome browser. If you perform the multi-factor (or 2FA, or TFA) authentication step inside of a Chrome browser, you can record the required steps to set up browser tests. 
+
+Some MFA providers might detect our Browser tests as bots and prevent them from logging in, for example, by adding a re-Captcha. If that is your case, consider reaching out to your MFA provider to see if it is possible to turn off bot detection for a specific set of credentials for testing purposes.
+
+If your MFA process involves steps performed outside of the browser, such as voice, text message, or opening a mobile application, consider reaching out to your MFA provider to ask if your MFA settings could be modified or if MFA could be turned off for a specific set of credentials for testing purposes.
+Depending on the type of MFA leveraged by your application, [JavaScript steps][3] could help to work around that.
+
+## Account security
+
+### Secure your authentication data
+
+Store your credentials as [global variables][4] (for example, one global variable for username, another one for password) and  set these variables as secure to obfuscate their values from anyone else who has access to your instance of Datadog:
+
+Once you create the secure variables, you can then import these global variables into your browser tests and leverage them for your login steps: 
```

Comment:
Can we add a link to /synthetics/browser_tests/actions#a-global-variable instead of the screenshot? To make sure they know which other doc page to look if they need more info on that?